### PR TITLE
https://github.com/atomic-penguin/sm-hidden/issues/13

### PR DESCRIPTION
**Pull request by ghost**
http://forums.alliedmods.net/showthread.php?t=99147
based on this plugin
it seems to use a global constant defined as (1<<11)
which we already had defined

### CODE REVIEW

File: hidden.sp [hunk 1]
```diff
@@ -409,6 +410,7 @@ public Action:player_death(Handle:event, const String:name[], bool:dontBroadcast
             if (hiddenHp>hiddenHpMax) {
                 hiddenHp=hiddenHpMax;
             }
+            SetEntProp(attacker, Prop_Send, "m_nPlayerCond", cond & ~HIDEHUD_BONUS_PROGRESS);
```
Comment (atomic-penguin) [inline hidden.sp:413]: Should this not be `~PLAYER_FIRSTBLOOD`, rather than `~HIDEHUD_BONUS_PROGRESS`?

**Comment by ghost** [comment]
in the plugin (1<<11) is defined as PLAYER_FIRSTBLOOD whereas we define it as HIDEHUD_BONUS_PROGRESS, i'm working on the assumption that it will function the same

**Comment by ghost** [comment]
// Player Condition  ----------------------------------------------------------
#define PLAYER_FIRSTBLOOD (1 << 11)

from ln.35

**Comment by ghost** [comment]
ok so do we think it will work if we also have a
#define PLAYER_FIRSTBLOOD (1 << 11)
?

**Pull request closed without merging**